benchmarks/buiter_et_al_2016_jsg/exp_2_low_resolution.prm

```diff
@@ -0,0 +1,199 @@
+#### This is the unstable wedge experiment 2
+
+###  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 14400
+set Use years in output instead of seconds = false
+set CFL number				   = 0.5
+set Maximum time step			   = 1e5 #3.6
+
+set Nonlinear solver scheme                = single Advection, iterated Stokes
```

Comment:
we should talk with @MFraters about trying out the Newton solver.

Reply:
We did a test for the Newton solver and it shows no convergence at the first timestep.

Reply:
what parameters did you use?

Reply:
@MFraters  we set Nonlinear solver scheme = iterated Advection and Newton Stokes

Reply:
> what parameters did you use?

we set Nonlinear solver scheme = iterated Advection and Newton Stokes